### PR TITLE
Completed Prep-Javascript-Functions

### PR DESCRIPTION
**Pull request by RicardoDav**
image

### CODE REVIEW

File: prep-javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+const addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log('Add two numbers exercise: ' + addTwoNumbersResult);
+function convertHourstoMinutes(hours) {
+  return hours * 60;
+}
+const Hours = convertHourstoMinutes(2);
```
Comment (BrettAlbright) [inline prep-javascript-functions/main.js]: you want to typically avoid uppercase variable names in javascript, there are typically only used for Classes and Constructor functions

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
code looks good! I would change the variables to starting with a lowercase letter

File: prep-javascript-functions/main.js (hunk 1 above)
Comment (RicardoDav) [reply prep-javascript-functions/main.js]: I pushed the fixes

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**